app/controllers/application_controller.rb

```diff
@@ -5,6 +5,159 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, 'yaaaaaaaaaaaaaaaaaaaaas_queeeeeeeeeeen'
   end
 
-end
\ No newline at end of file
+  # Fwitter Homepage
+  get '/' do
+    erb :index
+  end
+
+  ############# TWEETS #####################
+
+  # READ
+  get '/tweets' do #all tweets
```

Comment:
This controller is overloaded. Consider separating your concerns by splitting some of these actions into separate controllers (i.e. a users controller and a tweets controller).